docs/openshift-ai/other-projects/configure-jupyter-notebook-use-gpus-aiml-modeling.md

```diff
@@ -0,0 +1,377 @@
+# Configure a Jupyter notebook to use GPUs for AI/ML modeling
+
+**Prerequisites**:
+
+Prepare your Jupyter notebook server for using a GPU, you need to have:
+
+- Select the correct data science project and create workbench, see
+[Populate the data science project](../data-science-project/using-projects-the-rhoai.md#populate-the-data-science-project)
+for more information.
+
+Please ensure that you start your Jupyter notebook server with options as depicted
+in the following configuration screen. This screen provides you with the opportunity
+to select a notebook image and configure its options, including the number of GPUs.
+
+![PyTorch Workbech Information](images/pytorch-workbench.png)
+
+For our example project, let's name it "PyTorch Workbench". We'll select the
+**PyTorch** image, choose a **Deployment size** of **Small**, **Number of GPUs**
+as **1** and allocate a **Cluster storage** space of **1GB**.
+
+If this procedure is successful, you have started your Jupyter notebook server.
+When your workbench is ready, the state will change to *Running* and you can select
+"Open" to go to your environment:
+
+![Open JupyterLab Environment](images/open-jupyter-lab.png)
```

Comment:
this image is too small to see, save it as a larger file and/or circle the area you are hovering over in red.